Src/SwqlStudio/3rdParty/ScintillaNET-FindReplaceDialog/FindReplace/FindReplaceDialog.cs

```diff
@@ -1080,7 +1073,7 @@ private CharacterRange ReplaceNext(bool searchUp)
                 if (rdoRegexR.Checked)
                 {
                     rr = new Regex(txtFindR.Text, GetRegexOptions());
-                    string selRangeText = Scintilla.GetTextRange(selRange.cpMin, selRange.cpMax - selRange.cpMin + 1);
```

Comment:
Example of bad replace:
* query: `abc`
* search for regex: `b`
* replace with: `d`
* result was: `adcc`